README.md

```diff
@@ -18,7 +18,7 @@ When actively working on the content, the most useful option is the watch mode.
 npm run dev
 ```
 
-This will take a while the first time. The site will be created in the `academy` folder. Use `http-server` or something similar to view.
+This will take a while the first time. The site will be created in the `academy` folder. Use `http-server` or something similar to view. (In your browser you may need to open the academy folder if running from root.) HTML will usually auto-update as you save files.
```

Comment:
> HTML will usually auto-update as you save files.

I believe you'd still have to refresh on changes unless you use something like live server

Reply:
Oh, great point. I'll add that note real quick.